frontend/src/components/LabelledNumberInput.vue

```diff
@@ -2,10 +2,10 @@
 	<LabelledComponent :title="title" :id="id">
 		<slot></slot>
 		<div class="quantity">
-			<input ref="input" type="number" :name="title" :value="isNaN(value) ? '' : value" :min="min" :max="max" :step="step" inputmode="numeric" :id="title.toLowerCase().replaceAll(' ', '') + id" @change="change" :placeholder="placeholder" />
+			<input ref="input" type="number" :name="title" :value="isNaN(value) ? '' : value" :min="min" :max="max" :step="isSteppable ? 'any' : step" inputmode="numeric" :id="title.toLowerCase().replaceAll(' ', '') + id" @change="change" :placeholder="placeholder" />
```

Comment:
the min and max variables passed to the input should be the computed, not the prop!